package.json

```diff
@@ -11,7 +11,13 @@
     "build": "rollup --config",
     "commitmsg": "validate-commit-msg",
     "destroy": "rm -rf node_modules package-lock.json dist",
-    "deploy": "npm run destroy && npm install && npm run build && aws s3 cp ./dist/webloader.js  s3://evolv-execution-plan-binaries/asset-manager/releases/latest/webloader.js --acl public-read && aws s3 cp ./dist/webloader.min.js  s3://evolv-execution-plan-binaries/asset-manager/releases/latest/webloader.min.js --acl public-read"
+    "prerelease": "npm ci && npm run build && npm run deploy:version:s3 && npm run deploy:next:s3 && npm run deploy:next:npm",
+    "release": "npm ci && npm run build && npm run deploy:version:s3 && npm run deploy:latest:s3 && npm run deploy:latest:npm",
+    "deploy:next:s3": "aws s3 cp ./dist/webloader.js  s3://evolv-execution-plan-binaries/asset-manager/releases/next/webloader.js --acl public-read && aws s3 cp ./dist/webloader.min.js  s3://evolv-execution-plan-binaries/asset-manager/releases/next/webloader.min.js --acl public-read",
+    "deploy:version:s3": "aws s3 cp ./dist/webloader.js  s3://evolv-execution-plan-binaries/asset-manager/releases/$SEM_VER/webloader.js --acl public-read && aws s3 cp ./dist/webloader.min.js  s3://evolv-execution-plan-binaries/asset-manager/releases/$SEM_VER/webloader.min.js --acl public-read",
+    "deploy:latest:s3": "aws s3 cp ./dist/webloader.js  s3://evolv-execution-plan-binaries/asset-manager/releases/latest/webloader.js --acl public-read && aws s3 cp ./dist/webloader.min.js  s3://evolv-execution-plan-binaries/asset-manager/releases/latest/webloader.min.js --acl public-read",
+    "deploy:next:npm": "node ./scripts/updateVersion.js ./package.json $SEM_VER && npm publish --tag next --access public",
+    "deploy:latest:npm": "node ./scripts/updateVersion.js ./package.json $SEM_VER && npm publish --access public"
```

Comment:
NPM scripts can accept arguments in a way, so these feel like an appropriate place to be DRY by writing some helper scripts.

To pass arguments, you use the double dash `--` (whitespace before or after). Anything following the double dash will be sent along to the command specified for the script. Example
```
{
  "scripts" : {
    "print": "echo"
  }
}

$ npm run print -- "Hello world"
Hello world
```

Reply:
Bonus: there is a package named [commander](https://www.npmjs.com/package/commander) that helps with writing CLIs. Similar concept to `click` in Python.

```
const { program } = require('commander');
 
program
  .option('-d, --debug', 'output extra debugging')
  .option('-s, --small', 'small pizza size')
  .option('-p, --pizza-type <type>', 'flavour of pizza');
 
program.parse(process.argv);
 
if (program.debug) console.log(program.opts());
console.log('pizza details:');
if (program.small) console.log('- small pizza size');
if (program.pizzaType) console.log(`- ${program.pizzaType}`);
```